source/Symbol/SwiftASTContext.cpp

```diff
@@ -5090,8 +5090,32 @@ bool SwiftASTContext::IsSelfArchetypeType(const CompilerType &compiler_type) {
   return false;
 }
 
-bool SwiftASTContext::IsPossibleZeroSizeType(
-  const CompilerType &compiler_type) {
+static CompilerType BindAllArchetypes(CompilerType type,
```

Comment:
Maybe this can be made public in `SwiftASTContext`? 
Seems something could be of more general use.